underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -575,7 +565,11 @@ private ObjectPermissions getPermissionsInternal() {
 
   @Override
   protected String getRootKey() {
-    return mRootKey;
+    if (mUri.getScheme() != null && mUri.getScheme().equals("s3a")) {
```

Comment:
`"s3a".equals(mUri.getScheme())`?
Should we also check for `s3`, is there any risk of the uri scheme not being `s3a` or `s3`?

Reply:
- updated
- not really possible to be any other scheme rather than `s3a` or `s3`